page-content/Projects.html

```diff
@@ -4,5 +4,9 @@ <h3 class="page-title">Projects</h3>
     Here are all the projects that exist.
 </p>
 
+<div class="reload-button">
+<button class="btn btn-primary" type="submit" id="projects-reload-data-btn">Reload Data</button>
```

Comment:
Please indent this line, thanks.

Reply:
Resolved by commit 56a031797d7cf4ea3a150b0693d1b651bb606d09